packages/graph-explorer-proxy-server/node-server.js

```diff
@@ -117,20 +129,26 @@ dotenv.config({ path: "../graph-explorer/.env" });
       const requestSig = reqObjects[2];
 
       await getAuthHeaders("gremlin", req, endpoint_url, requestSig);
-
+      
       const response = await fetch(
         `${endpoint}/?gremlin=` + 
         encodeURIComponent(req.query.gremlin),
         {
           headers: req.headers,
         }
       );
-
+      
+      if (!response.ok){
+        let msg = `Error getting response from gremlin(default) endpoint. [${response.statusText}] with query [${req.query.gremlin}].`
```

Comment:
What does gremlin(default) mean here? Otherwise, suggest rephrase to `Error getting response from Gremlin Server endpoint...`